docs/source/conf.py

```diff
@@ -61,10 +61,10 @@
 # built documents.
 #
 # The short X.Y version.
-version = '2.0'
+version = '2.2'
 
 # The full version, including alpha/beta/rc tags.
-release = '2.0'
+release = '2.2'
```

Comment:
noticed in setup.py you did 2.2.0 for the version, i don't think it impacts anything though